src/components/Restoration/CorrectCompanyName.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <v-form id="correct-name-form" ref="correctNameForm" v-model="valid" lazy-validation>
+    <v-row no-gutters>
+      <v-col>
+        <v-text-field
+          v-model="companyName"
+          id="company-name-input"
+          class="mb-n3"
+          filled
+          :rules="companyNameRules"
+        />
+      </v-col>
+    </v-row>
+  </v-form>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { CommonMixin } from '@/mixins/'
+import { ActionBindingIF, NameRequestIF } from '@/interfaces/'
+import { NameChangeOptions } from '@/enums/'
+
+@Component({
+  mixins: [
+    CommonMixin
+  ]
+})
+export default class CorrectCompanyName extends Vue {
+  /** Form Submission Prop */
+  @Prop({ default: null }) readonly formType!: NameChangeOptions
+
+  @Action setNameRequest!: ActionBindingIF
+
+  @Getter getNameRequestLegalName!: string
+  @Getter getNameRequest!: NameRequestIF
```

Comment:
Ideally, this low-level component wouldn't talk to the store, but would use props and events to communicate with its parent.

Or we could use only props for data in, and for functions in that get called to update something. In the parent, these functions would be references to the store methods or whatever.

Reply:
As we've discussed, I'm concerned about code duplication.  The more we copy code from one repro to another, the more we create a mess we'll have to clean up later.  I think it might be wise to raise this with the business to make sure they're aware of the issue and get approval before proceeding.

Reply:
Yep, will do.

To move these sub-components to the shared repo, I first have to refactor them slightly to use props/events instead of talking to the store directly. Then I'll have to go back to Edit UI and update their usage there. It's 1-2 days of work.

Reply:
As discussed at Standup this morning, I will make the duplicate components shared.